src/components/Episodes/Episodes.jsx

```diff
@@ -9,7 +9,18 @@ import {
 
 export default function Episodes(){
 
-	const episodes = data.map(RENDER.episode);
+	const episodes = data.map( (obj, index) => {
```

Comment:
only just realised this was an arrow inside a `.map` - I swear we fixed this - was it another branch? Can we restore this to `data.map(RENDER.episode)` and just do all of this logic inside that util? It's what the fellah is there for 🙏

Reply:
Nah, it's been like that for a bit.
Some of the changes requested are hidden for some reason, so might be that I missed it in previous pr.